dragonfly/world/axis.go

```diff
@@ -39,3 +39,7 @@ func (a Axis) FromString(s string) (interface{}, error) {
 	}
 	return nil, fmt.Errorf("unexpected axis '%v', expecting one of 'x', 'y' or 'z'", s)
 }
+
+func AllAxis() []Axis {
```

Comment:
Should probably name this `AllAxes` (and don't forget to document this).

Reply:
Renamed 👍